svo (actions dictionary).xml

```diff
@@ -1772,6 +1772,31 @@ if not next(svo.dict) then
         oncompleted = function() svo.rmaff('justice') end,
       }
     },
+    pyre = {
+      herb = {
+        isadvisable = function()
+          return (affs.pyre and (not (svo.haveskillset('discipline') or svo.haveskillset('malignity') or svo.haveskillset('valour') or svo.haveskillset('dominion')) or not svo.dict.rage.misc.isadvisable())) or false
```

Comment:
Not 100% sure here but will it work for classes that don't have rage?

Reply:
This is strange, I don't understand those isadvisable checks. Why is it checking if the user has either of those knight skills? And uh.. Rage? I don't think the affliction only affects knights or interferes with rage? Either this one or the justice above? My guess is that whoever wrote this before mistook if for a skill maybe?

Im probably being stupid lol

Reply:
@TheLastDarkthorne I didn't remember what it did at first too, but in justice and other rage-healable affs it's just a safety check to ensure that both herb and rage cures don't go off at the same time. Like @veles-achaea mentions we need to verify if rage can cure this aff too

Reply:
There's a few things that could be done here, but what are you looking to do exactly? Remind people to reset their prios to default for the new thing?